Makefile

```diff
@@ -102,8 +102,17 @@ eunit: couch
 
 .PHONY: javascript
 # target: javascript - Run JavaScript test suites or specific ones defined by suites option
-javascript: all share/www/script/test
```

Comment:
@kxepal sorry I had to undo your cleanup here, because we are depending on different dirs, depending on whether we run the JS tests from the source tree or a release tarball. I don’t know how to better formulate this in make.